.github/workflows/coverage.yml

```diff
@@ -14,13 +14,16 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install Rust
-        run: rustup update stable
+        run: rustup install nightly && rustup default nightly
```

Comment:
Just curious - why does coverage require nightly now?

Reply:
It's needed to merge the doctest coverage with the normal test coverage (don't know why it's not in stable). This little dance is an unfortunate side effect of `nextest` not running the doctest (yet).